src/services/accountService.ts

```diff
@@ -25,7 +29,7 @@ export async function getAccount(network: string, address: string) {
 	const extrinsic = await getExtrinsic(network, filter);
 
 	if (!extrinsic) {
-		return null;
+		return { address: address, id: address };
```

Comment:
Ehm no, dava ti tohle smysl? Koukni na celou tu `getAccount` funkci, neni zbytecny tahat ty extrisicy kdyz to nic neovlivni?